<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TcpClient</class>
 <widget class="QMainWindow" name="TcpClient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>405</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>405</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>405</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>TcpClient 微信公众号（智慧小巷）</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>153</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>网络配置</string>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>连接IP</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="ip_edit"/>
     </item>
     <item>
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>端口号</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLineEdit" name="prot_edit"/>
     </item>
     <item>
      <widget class="QPushButton" name="btn_connect">
       <property name="text">
        <string>连接</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ckb_hex_show">
       <property name="text">
        <string>HEX显示</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QCheckBox" name="ckb_hex_send">
       <property name="text">
        <string>HEX发送</string>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="clear_btn">
       <property name="text">
        <string>清空窗口</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>270</y>
      <width>532</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>发送</string>
    </property>
    <widget class="QTextEdit" name="send_edit">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>371</width>
       <height>71</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="send_btn">
     <property name="geometry">
      <rect>
       <x>390</x>
       <y>20</y>
       <width>111</width>
       <height>71</height>
      </rect>
     </property>
     <property name="text">
      <string>发送</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>171</x>
      <y>9</y>
      <width>361</width>
      <height>251</height>
     </rect>
    </property>
    <property name="title">
     <string>接收</string>
    </property>
    <layout class="QGridLayout" name="gridLayout_2">
     <item row="0" column="0">
      <widget class="QTextEdit" name="re_edit"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
